bot_jobs/build_province_deaths.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import datetime
+import json
+
+DEATH_DATA_14DAYS_OUT_PATH = "../components/gis/data/province-deaths-data-14days.json"
```

Comment:
I think we should combine it to province-data-14days.json

Reply:
Combined in latest commit, code maybe a bit messy.